src/components/data/LocalVariablesContent.js

```diff
@@ -47,25 +70,27 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
     setEditIndex()
     setEditKey()
     setEditValue()
+    setEditDescription()
     setEditType()
   }
 
-  const initialAttributes = _.flow(
-    _.toPairs,
-    _.remove(([key]) => /^description$|:|^referenceData_/.test(key))
-  )(attributes)
+  const toDescriptionKey = k => DESCRIPTION_PREFIX + k
+
+  const initialAttributes = convertInitialAttributes(attributes)
 
   const creatingNewVariable = editIndex === initialAttributes.length
   const amendedAttributes = _.flow(
-    _.filter(([key, value]) => Utils.textMatch(textFilter, `${key} ${value}`)),
-    _.sortBy(_.first),
-    arr => [...arr, ...(creatingNewVariable ? [['', '']] : [])]
+    _.filter(([key, value, description]) => Utils.textMatch(textFilter, `${key} ${value} ${description}`)),
+    arr => [...arr, ...(creatingNewVariable ? [['', '', '']] : [])]
```

Comment:
Just noticing, I think this could actually be
```suggestion
    creatingNewVariable ? Utils.append(['', '', '']) : _.identity
```
But I don't know if that's actually easier to read, so it's your call!

Reply:
Done.